coffee/lib/abstract-chosen.coffee

```diff
@@ -55,20 +54,12 @@ class AbstractChosen
     else
       item.html
 
-  mouse_enter: -> @mouse_on_container = true
-  mouse_leave: -> @mouse_on_container = false
-
   input_focus: (evt) ->
     if @is_multiple
       setTimeout (=> this.container_mousedown()), 50 unless @active_field
     else
       @activate_field() unless @active_field
 
-  input_blur: (evt) ->
-    if not @mouse_on_container
-      @active_field = false
-      setTimeout (=> this.blur_test()), 100
-
```

Comment:
Since we now only trigger our `chosen:focus` and `chosen:blur` events when focus is lost on the entire container or its descendants, we no longer need to track mouse positioning.